opentech/static_src/src/sass/apply/components/_activity-feed.scss

```diff
@@ -0,0 +1,40 @@
+.activity-feed {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: -1;
+    opacity: 0;
+    transition: opacity, z-index, $transition;
+
+    &.is-open {
+        z-index: 20;
+        width: 100%;
+        height: 100vh;
+        max-height: 100%;
+        overflow: scroll;
+        background: $color--white;
+        opacity: 1;
+    }
+
+    &__header {
+        padding: 0 20px;
+        background-color: $color--default;
+
+        @include media-query(tablet-portrait) {
+            padding: 0;
```

Comment:
This padding removal means the close icon is right up against the window edge until the wrapper--medium max-width applies at 1020px. screen grab attached.

![otf-dev torchboxapps com_apply_submissions_](https://user-images.githubusercontent.com/35495969/37039072-73a1cab2-214e-11e8-9689-61c4794707fb.png)

Reply:
Good spot - addressed here: https://github.com/OpenTechFund/opentech.fund/pull/135/commits/886dc8015b458ea87bb0d14075e50995d1eeb4bc